pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,10 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			err := ValidateInsecureEdgeTerminationPolicy(obj)
```

Comment:
doh. Now that I actually fixed this import, it requires conversion from v1 to internal in order for the validation function to work. Should I proceed with converting just to validate, duplicate the validation function to work against the versioned object, or isn't this validation step really necessary?